<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentDialog</class>
 <widget class="QStackedWidget" name="PaymentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pembayaran</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="page">
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bayar (Enter)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Batal (ESC)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPaymentType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipe Pembayaran: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblErrorState">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPaymentType">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="SpinBoxPayment" name="txtAmount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="prefix">
     <string>Rp </string>
    </property>
    <property name="maximum">
     <number>10000000</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_2">
   <widget class="QLabel" name="lblChange">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tutup</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBoxPayment</class>
   <extends>QSpinBox</extends>
   <header>spinboxpayment.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
